velox/functions/prestosql/Probability.h

```diff
@@ -94,6 +94,30 @@ struct BinomialCDFFunction {
     boost::math::binomial_distribution<> dist(numOfTrials, successProb);
     result = boost::math::cdf(dist, value);
   }
+
+  FOLLY_ALWAYS_INLINE void
```

Comment:
It seems like a lot of this code is duplicated with the earlier call. 

It might be better to write a common function where the value parameter is a templated type. And invoke that function in these call methods with the required type for it.

Reply:
@aditi-pandit As we are planning to remove this function signature (double, bigint) in a later PR, do we need to write a common function here?

Reply:
Made the changes.